src/services/lock/service/lock.service.ts

```diff
@@ -118,131 +102,35 @@ export class LockService {
     });
 
     this.#logger.log(`This node is currently registered as lock keeper for ${id}`);
-
     this.registeredLocks[id] = {};
 
     return lock;
   }
 
-  /**
-   * Unregisters a previously acquired lock, making it available for other instances.
-   * @param {string} id - The unique identifier of the lock to be unregistered.
-   */
   async unregisterLock(id: string): Promise<void> {
```

Comment:
Function comments are fine, but extra types are not necessary

```suggestion
  /**
   * Unregisters a previously acquired lock, making it available for other instances.
   * @param id - The unique identifier of the lock to be unregistered.
   */
  async unregisterLock(id: string): Promise<void> {
```